### PR TITLE
Issue #152: SuppressionPatchXpathFilter: DefaultComesLast's context strategy

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #152: SuppressionPatchXpathFilter: DefaultComesLast's context strategy

common pattern https://github.com/checkstyle/patch-filters/issues/8#issuecomment-665565892

```
             default:  // violation context
                 break;
+            case 3:
+                break;
         }

```

the default should be after all the cases in a switch statement, and `case 3` is not `default`'s child nodes.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java [hunk 1]
```diff
@@ -157,4 +157,12 @@ public void testEmptyCatchBlock() throws Exception {
         testByConfig("EmptyCatchBlock/patchedline/defaultContextConfig.xml");
         testByConfig("EmptyCatchBlock/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/152")
```
Comment (rdiachenko) [inline src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java:162]: Why is it ignored?

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DefaultComesLast/context/defaultContext.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git a/Test.java b/Test.java
+index be9c189..061cde2 100644
+--- a/Test.java
++++ b/Test.java
+@@ -9,6 +9,8 @@ public class Test {
+                 break;
+             default:  // violation context
```
Comment (HuGanghui) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DefaultComesLast/context/defaultContext.patch:7]: the default should be after all the cases in a switch statement, and `case 3` is not `default`'s child nodes.

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java (hunk 1 above)
Comment (HuGanghui) [reply src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java:162]: https://github.com/checkstyle/patch-filters/pull/154#discussion_r461371947

Comment (rdiachenko) [reply src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java]: @HuGanghui please add this case to the corresponding group of patterns if you haven't done it yet.

**Pull request merged**